config/config.go

```diff
@@ -84,4 +96,7 @@ func AddFlags(cmd *cobra.Command) {
 	cmd.Flags().BytesHex(flagNamespaceID, def.NamespaceID[:], "namespace identifies (8 bytes in hex)")
 	cmd.Flags().Bool(flagFraudProofs, def.FraudProofs, "enable fraud proofs (experimental & insecure)")
 	cmd.Flags().Bool(flagLight, def.Light, "run light client")
+	cmd.Flags().String(flagSequencerListenAddress, def.SequencerListenAddress, "listening address for centralized sequencer")
```

Comment:
may be expand the flag description to indicate if there is any format for the listen addr?

Reply:
removed the flag entirely